src/Selenium2Library/keywords/cookie.py

```diff
@@ -1,46 +1,58 @@
-from .keywordgroup import KeywordGroup
+from Selenium2Library.robotlibcore import keyword
 
 
-class CookieKeywords(KeywordGroup):
+class CookieKeywords(object):
```

Comment:
Good point. I'd say all lib components should extend same base class. Consistency FTW!